entry.sh

```diff
@@ -1,8 +1,8 @@
 # In cmd.exe
-venv\Scripts\activate.bat
+.venv/Scripts/activate.bat
 # In PowerShell
-venv\Scripts\Activate.ps1
+.venv/Scripts/Activate.ps1
 # In Mac/Linux
-source .venv/bin/activate
```

Comment:
This line is still needed in for when you create a env file using a unix system, just add it back using /bin/

Reply:
yeah i know i changed it so that it didnt give me errors but it didnt really matter